<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="OperationModeBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string> Режим работы</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>211</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Режим</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modeCmbx"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Скорость</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="speedCmbx"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Тип связи</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ComunicationTypeCmbx"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Частота</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="FrequencySbx">
          <property name="maximum">
           <number>30000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="inversionCbx">
        <property name="text">
         <string>Инверсия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compressionCbx">
        <property name="text">
         <string>Сжатие</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="StateGbx">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Состояние</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>211</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="isValidModemcbx">
        <property name="text">
         <string>Модем исправен</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isInfoAvaliableCbx">
        <property name="text">
         <string>Наличие информации</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="Signallabel">
          <property name="text">
           <string>Сигнал:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="signalCmbx"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="connectLabel">
          <property name="text">
           <string>Связь:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="connectionCmbx"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="SignalLevellbl">
          <property name="text">
           <string>Уровень сигнала:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="levelSignalSpinbox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>511</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QLabel" name="statusUDPlabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Статус UDP:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Просмотр 
журнала</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
